enterprise/reporting/flusher/flusher.go

```diff
@@ -303,6 +315,23 @@ func (f *Flusher) delete(ctx context.Context, minReportedAt, maxReportedAt time.
 	return err
 }
 
+func (f *Flusher) vacuum(ctx context.Context) error {
+	vacuumStart := time.Now()
+	var sizeEstimate int64
+	if err := f.db.QueryRowContext(
+		ctx, fmt.Sprintf(`SELECT pg_table_size(oid) from pg_class where relname='%s'`, f.table),
```

Comment:
We can use the parameterized query here.
```suggestion
		ctx, `SELECT pg_table_size(oid) from pg_class where relname = $1`, f.table,
```